frontend/src/pages/clusterSettings/const.ts

```diff
@@ -12,9 +12,16 @@ export const MAX_MINUTE = 59;
 export const MIN_MINUTE = 0;
 export const MIN_CULLER_TIMEOUT = 600; // 10 minutes
 export const DEFAULT_CULLER_TIMEOUT = 31536000; // 1 year as no culling
-export const DEFAULT_CONFIG: ClusterSettings = {
+export const DEFAULT_CONFIG: ClusterSettingsType = {
   pvcSize: DEFAULT_PVC_SIZE,
   cullerTimeout: DEFAULT_CULLER_TIMEOUT,
   userTrackingEnabled: false,
   notebookTolerationSettings: null,
 };
+export const PVC_SIZE_HINT = 'Note: PVC size must be between 1 GiB and 16384 GiB.';
+export const CULLER_TIMEOUT_HINT =
+  'Note: Notebook culler timeout must be between 10 minutes and 1000 hours.';
+export const DEFAULT_TOLERATION_VALUE = 'NotebooksOnly';
+export const TOLERATION_FORMAT = /^([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9]$/;
+export const TOLERATION_FORMAT_ERROR =
```

Comment:
The `PVC_SIZE_HINT`, `CULLER_TIMEOUT_HINT` and `TOLERATION_FORMAT_ERROR` are used both in the page and in the storybook file to find the element. The others are not, I guess it was because they were set as the const before so I moved them to the file simply, I can make them inline if it's needed.

Reply:
no need. i didnt know they were already consts